src/facilities/costCenters.spec.js

```diff
@@ -0,0 +1,548 @@
+          .stub(facilitiesUtils, 'formatCostCenterFromServer')
+          .returns(formattedCostCenterFromServer);
+        formatCostCenterToServer = this.sandbox
+          .stub(facilitiesUtils, 'formatCostCenterToServer')
+          .returns(formattedCostCenterToServer);
+        request = {
+          ...baseRequest,
+          post: this.sandbox.stub().resolves(formattedCostCenterFromServer)
+        };
+
+        const costCenters = new CostCenters(baseSdk, request, expectedHost);
+
+        promise = costCenters.create(initialCostCenter);
+      });
+
+      it('formats the submitted cost center object to send to the server', function() {
+        expect(formatCostCenterToServer).to.be.calledWith(initialCostCenter);
+      });
+
+      it('creates a new cost center', function() {
+        expect(request.post).to.be.calledWith(
+          `${expectedHost}/costcenters`,
+          formattedCostCenterToServer
+        );
+      });
+
+      it('formats the returned cost center object', function() {
+        return promise.then(() => {
+          expect(formatCostCenterFromServer).to.be.calledWith(
+            formattedCostCenterFromServer
+          );
+        });
+      });
+
+      it('returns a fulfilled promise with the new cost center information', function() {
+        return expect(promise).to.be.fulfilled.and.to.eventually.equal(
+          formattedCostCenterFromServer
+        );
+      });
+    });
+
+    context('when there is missing required information', function() {
+      ['name', 'organizationId'].forEach(function(field) {
+        it(`it throws an error when ${field} is missing`, function() {
+          const costCenter = fixture.build('costCenter');
+          const costCenters = new CostCenters(
+            baseSdk,
+            baseRequest,
+            expectedHost
+          );
+          const promise = costCenters.create(omit(costCenter, [field]));
+
+          return expect(promise).to.be.rejectedWith(
+            `A ${field} is required to create a new cost center.`
+          );
+        });
+      });
+    });
+  });
+
+  describe('delete', function() {
+    context('when all required information is supplied', function() {
+      let expectedCostCenterId;
+      let promise;
+
+      beforeEach(function() {
+        expectedCostCenterId = fixture.build('costCenter').id;
+
+        const costCenters = new CostCenters(baseSdk, baseRequest, expectedHost);
+        promise = costCenters.delete(expectedCostCenterId);
+      });
+
+      it('requests to delete the cost center', function() {
+        expect(baseRequest.delete).to.be.calledWith(
+          `${expectedHost}/costcenters/${expectedCostCenterId}`
+        );
+      });
+
+      it('returns a fulfilled promise', function() {
+        return expect(promise).to.be.fulfilled;
+      });
+    });
+
+    context('when there is missing required information', function() {
+      it(`it throws an error when the cost center id is missing`, function() {
```

Comment:
I should have noticed this before, but there is one too many its here and should have single quotes